README.md

```diff
@@ -0,0 +1,56 @@
+# Azure SDK for Python
+
+[![Python](https://img.shields.io/pypi/pyversions/azure-core.svg?maxAge=2592000)](https://pypi.python.org/pypi/azure/) [![BuildStatus](https://dev.azure.com/azure-sdk/public/_apis/build/status/46?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=46&branchName=master) [![Build](https://travis-ci.org/Azure/azure-sdk-for-python.svg?branch=master)](https://travis-ci.org/Azure/azure-sdk-for-python) [![Dependencies](https://img.shields.io/badge/dependencies-analyzed-blue.svg)](https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-python/dependencies/dependencies.html)
+
+This repository contains official Python libraries for Azure services. For documentation go to [Azure SDK for Python documentation](http://aka.ms/python-docs).
+
+You can find a complete list of all the packages for these libraries [here](./packages.md).
+
+## Getting started
+
+For your convenience, each service has a separate set of libraries that you can choose to use instead of one, large Azure package. To get started with a specific library, see the `README.md` (or `README.rst`) file located in the library's project folder.
+
+You can find service libraries in the `/sdk` directory.
+
+### Prerequisites
+The client libraries are supported on Python 2.7 and 3.5.3 or later.
+
+## Packages available
+Each service might have a number of libraries available from each of the following categories:
+* [Client - July 2019 Preview](#Client-July-2019-Preview)
+* [Client - Stable](#Client-Stable)
+* [Management](#Management)
+
+
+### Client: July 2019 Preview
+New wave of packages that we are currently releasing in **Preview**. These libraries allow you to use and consume existing resources and interact with them, for example: upload a blob. These libraries share a number of core functionalities such as: retries, logging, transport protocols, authentication protocols, etc. that can be found in the [azure-core](./sdk/core/azure-core) library. You can learn more about these libraries by reading guidelines that they follow [here](https://azuresdkspecs.z5.web.core.windows.net/PythonSpec.html).
+
+The libraries released in July preview:
+
+- [azure-cosmos](./sdk/cosmos/azure-cosmos)
+- [azure-eventhubs](./sdk/eventhub/azure-eventhubs)
+- [azure-keyvault-keys](./sdk/keyvault/azure-keyvault-keys)
+- [azure-keyvault-secrets](./sdk/keyvault/azure-keyvault-secrets)
+- [azure-identity](./sdk/identity/azure-identity)
```

Comment:
I note identity is included here. I was asked if we should include it in Java, but I decided not to because it wasn't in the .net readme. We should be consistent here.

Reply:
We should include `azure-identity`. This is something that users will directly import/reference and use in their applications.